### ISSUE TITLE
Vim fails to source `defaults.vim`?

### ISSUE THREAD

**Issue by sincere267** [vim/vim-appimage#58]
```bash
$ ./Vim-v9.0.1885.glibc2.29-x86_64.AppImage
E1187: Failed to source defaults.vim
Press ENTER or type command to continue
```

**Comment by chrisbra** [comment]
duplicate of #55 should be fixed by #56

**Comment by sincere267** [comment]
OK, it has been fixed in `Vim-v9.0.1894.glibc2.29-x86_64.AppImage`.
